lib/api.ts

```diff
@@ -4,12 +4,15 @@ import { Activity, ActivityData, Contributor, Highlights } from "./types";
 import { padZero } from "./utils";
 import { readFile, readdir } from "fs/promises";
 import { existsSync } from "fs";
+import { fetchGithubDiscussionForUser } from "@/lib/discussion";
+import { ParsedDiscussion } from "@/scraper/src/github-scraper/types";
+import { env } from "@/env.mjs";
 
 const root = join(process.cwd(), "data-repo/contributors");
 const slackRoot = join(process.cwd(), "data-repo/data/slack");
 const githubRoot = join(process.cwd(), "data-repo/data/github");
 
-const points = {
+const points: { [key: string]: number } = {
```

Comment:
A cleaner more safer way is to use `as const` in the end. It'll have much more stricter type safety! 

Ref: https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-4.html#const-assertions


```js
const foo = {
  a: 'hello',
  b: 'world',
} as const;
```

```suggestion
const points = {
```

Reply:
![image](https://github.com/user-attachments/assets/20931c45-7720-4f7f-b14d-a39bc8893bc3)

If i remove types from there than generate this error so the type declaration is require there

Reply:
The correct way to solve that is to have correct type definition for activity.type instead.